concepts/80 Troubleshooting/00 Basics/15 Examine CSS.md

```diff
@@ -0,0 +1,23 @@
+DevExtreme components consist of HTML elements and obey CSS rules. If the component is not displayed correctly, examine its CSS.
+
+### Issue 
+
+The [DataGrid](/Documentation/ApiReference/UI_Components/dxDataGrid/) colorizes its [toolbar](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/toolbar/) without any colorizing properties specified.
+
+### Solution  
+
+Inspect the toolbar element. Right-click the element and choose the **Inspect** menu item or press Ctrl+Shift+C (Chrome) and click the element. Both actions navigate you to this element in the Elements tab of your browser's DevTools.
+
+Open the **Computed** tab and check the `background-color` CSS property.
+
+![Inspect CSS: the Computed tab opened](/images/Troubleshooting/troubleshooting-inspect-css-1.png)
+
+If this property does not exist, inspect the parent element.
+
+![Inspect CSS: parent element](/images/Troubleshooting/troubleshooting-inspect-css-2.png)
+
+The parent element has the `background-color` property. Expand it to see that the unexpected color comes from a rule for `.dx-toolbar` class in the *styles.css* file.
+
+![Inspect CSS: the `background-color` property expanded](/images/Troubleshooting/troubleshooting-inspect-css-3.png)
+
+Use a more specific CSS selector for this style to take effect only on the standalone toolbar. For example, assign an ID to the grid  toolbar and specify the following selector: `#my-toolbar .dx-toolbar`.
```

Comment:
I don't quite understand this section. 

I didn't want to change the background color for toolbar items. In other words, I probably have default settings specified in the DataGrid. Now all of a sudden the background color changes. 

I would assume that someone added a CSS style with the same name as used in my toolbar. So the more obvious "solution" would be not to add custom styles that conflict with default styles. 

If I want **custom** colors in my toolbar, then I can do things like "#my-toolbar .dx-toolbar". But if I want **default** colors, this the proposed solution seems counter-intuitive.

Reply:
I rewrote the "Issue" of this section. I think the problem was in its description. I hope now everything makes sense